### PR TITLE
Cargo and borgo stuffs. [change check a whole week overdue!]

### PR DESCRIPTION
**Pull request by Verkister**
-Added Eggnerd limb design disk to the mail order pack.
-Added the non-security oties on the pred crate picklist.
-Added mail order guard puppy crate. (the frankensteinian voremobcode+followerpetcode+beepskycode combowombo)
-Attempts a workaround for janihound belly not bulging from people. (I swear it was working before the merged code cleanup. Might've been just a random bug on the server tho since I haven't seen or heard further reliable occurences since. Better safe than sorry I guess :v)
-Janihound sprites centered properly.
-Made remains(mouse skeletons etc.) edible again.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/largecrate_vr.dm [hunk 1]
```diff
@@ -53,6 +53,8 @@
 						/mob/living/simple_animal/hostile/bear;0.5,
 						/mob/living/simple_animal/hostile/bear/brown;0.5,
 						/mob/living/simple_animal/hostile/carp,
+						/mob/living/simple_animal/otie/fluff,
```
Comment (SpadesNeil) [inline code/game/objects/structures/crates_lockers/largecrate_vr.dm]: otie/fluff is just an otie

You might want to fix this.

Comment (Verkister) [inline code/game/objects/structures/crates_lockers/largecrate_vr.dm]: It's the bare one without accessories that can still be tamed.

File: code/game/objects/structures/crates_lockers/largecrate_vr.dm [hunk 2]
```diff
@@ -53,8 +53,8 @@
 						/mob/living/simple_animal/hostile/bear;0.5,
 						/mob/living/simple_animal/hostile/bear/brown;0.5,
 						/mob/living/simple_animal/hostile/carp,
-						/mob/living/simple_animal/otie/fluff,
-						/mob/living/simple_animal/otie/fluff/cotie,
+						/mob/living/simple_animal/otie/fluff;0.5,
```
Comment (SpadesNeil) [inline code/game/objects/structures/crates_lockers/largecrate_vr.dm]: This just looks MORE incorrect.

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 3]
```diff
@@ -43,23 +43,23 @@
 	var/mob/living/carbon/human/friend
 	var/tamed = 0
 
-/mob/living/simple_animal/otie/fluff
+/mob/living/simple_animal/otie/fluff //gets the pet2tame feature. starts out hostile tho so get gamblin'
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Oh I see what is going on here. /fluff directories should not be handled like this. Fluff denotes a custom version of something, usually for a specific player.

Comment (Verkister) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Yeah should've probably changed it. Just sort of based it on the Runtime(the cat) befriending code that used the term.

**Comment by SpadesNeil** [comment]
i'm lazy reeeeeeeeeeeeeeeeeeeeeeee

**Comment by SpadesNeil** [comment]
Psst can you do me a favor?

I need a blue colorscheme of the otie to match the outdoors.
deadtrees

I want to throw "wild oties" out into the map for players to try hunting.

**Comment by SpadesNeil** [comment]
More mobs in general would be nice as I have desires to massively expand the outside world of the station for exploring.

**Comment by SpadesNeil** [comment]
Otie sprites have some directional issues. None of the laying sprites only face north, as north is just a copy of south, and the default otie full sprite only faces east. Pls fix.

**Comment by Verkister** [comment]
Alright

**Comment by Verkister** [comment]
All done. icon_state "siftusian" should point the code towards the new sprites.

**Pull request merged**